ui/src/stores/doc.js

```diff
@@ -70,13 +78,16 @@ export default {
         pageMetadata: (state) => {
             return state.pageMetadata;
         },
-        resourceUrl: (state) => (path) => {
+        resourceUrl: (state) => (path, domain = "docs") => {
```

Comment:
NIT: since this is the intended domain anyway why not add the /

```suggestion
        resourceUrl: (state) => (path, domain = "/docs") => {
```

Reply:
I thought about it, it's just that I wanted to showcase that you can omit it but whatever :+1: